### DISCUSSION TITLE
deploying-asp-net-core-applications-to-kubernetes-part-1-an-introduction-to-kubernetes/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#4] [Announcements]
# deploying-asp-net-core-applications-to-kubernetes-part-1-an-introduction-to-kubernetes/

In this post I describe some of the basic concepts that you'll need to be familiar with to deploy ASP.NET Core applications to Kubernetes.

https://andrewlock.net/deploying-asp-net-core-applications-to-kubernetes-part-1-an-introduction-to-kubernetes/

**Comment by jonnochoo** [comment]
Thanks Andrew, this was really helpful to understand the different concepts of deployment, services, ingress etc. The diagrams helped a lot. Appreciate the effort

**Comment by khalidative** [comment]
The best explanation of the kubernetes overview I've seen yet. Thanks alot!

[a comment by an automated account is not shown]

**Reply by andrewlock** [reply]
<em>[Sock](https://disqus.com/by/andrewlockdotnet/) (@andrewlock) commented [on Disqus](https://disqus.com/home/forum/andrewlock/) at September 01 2020, 11:00, in reply to [Алексей Соснин](https://disqus.com/by/disqus_ObGVX3ipMM/)'s [comment](https://github.com/andrewlock/blog-comments/discussions/4#discussioncomment-6059506)</em>

---

<p>I have a post on probes coming up in this series :)</p><p>But in general, ASP.NET Core has health checks built in. Check out <a href="https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks" rel="nofollow noopener" title="https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks">https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks</a> for a huge number of possible checks!</p>

[a comment by an automated account is not shown]

**Comment by akhanalcs** [comment]
Can you please create a blog series on using Project Tye with Kubernetes as you mentioned at the start of this post? Thank you!

**Comment by akhanalcs** [comment]
Loved how nicely this is written. Thank you!

**Comment by aviv-pierrekore** [comment]
Great Blog .Thank you for sharing your experiences
